vagrant/UPGRADING.md

```diff
@@ -0,0 +1,49 @@
+UPGRADING
+=========
+
+0.1.0
+-----
+This adds the possibility to have a whole hiera hierarchy.
```

Comment:
So @Ma27 should at least provide some link or information about hiera in the README.md or in a real documentation